homeassistant/components/openhome/__init__.py

```diff
@@ -1 +1,62 @@
 """The openhome component."""
+
+import asyncio
+import logging
+
+import aiohttp
+from async_upnp_client.client import UpnpError
+from openhomedevice.device import Device
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = [Platform.MEDIA_PLAYER]
+
+CONFIG_SCHEMA = cv.empty_config_schema(DOMAIN)
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the component."""
+    hass.data[DOMAIN] = {}
+    return True
+
+
```

Comment:
This is not necessary. Just initialize it in `async_setup_entry` when necessary. 
```suggestion
```